Dockerfile

```diff
@@ -0,0 +1,76 @@
+FROM quay.io/aperture147/ubuntu:focal
+
+LABEL maintainer="Aperture <aperture147@gmail.com>"
+
+# Bypass tzdata config
+ENV TZ=UTC
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+ARG with_mysql=true
+ARG with_sqlite3=false
+ARG with_pgsql=false
+ARG with_odbc=false
+ARG with_lua=false
```

Comment:
Should lua support enable as default ?

Reply:
In my build, enable LUA will cause pvpgn server to crash everytime an user connect. Btw LUA is a experimental feature so I don't think we should enable this by default.

Reply:
> Should lua support enable as default ?

Sorry, for the false claim, server crashing is not related to Lua since it happens on any build configuration if you build on branch `master`